src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -116,21 +116,11 @@ export default class Fms {
         */
         this._previousRouteSegments = [];
 
-        // TODO: Use this!
-        /**
-         * Airport the aircraft arrives to
-         *
-         * @for Fms
-         * @property arrivalAirport
-         * @type {AirportModel}
-         */
-        this.arrivalAirport = null;
-
```

Comment:
in this case here you have identified a problem; we are currently storing `RunwayModels`, and you would like to store `AirportModels` as well.  The airports themselves contain the `RunwayModels`.  

I agree in that we should try to remove the reliance on `airport_get()`, but I think there is a better way.  Storing the `AirportModel` (which is huge) times two for arrival and departure seems very heavy handed to me.  I would really like to see what you have planned with this, perhaps there is a better way?  Perhaps we add a facade to the `NavigationLibrary` or something?